inference-engine/src/vpu/graph_transformer/src/stages/interp.cpp

```diff
@@ -80,8 +89,19 @@ void FrontEnd::parseInterp(const Model& model, const ie::CNNLayerPtr& layer, con
     VPU_THROW_UNLESS(outputs.size() == 1,
                      "Interp stage with name {} must have only 1 output, "
                      "actually provided {}", layer->name, outputs.size());
+    ie::details::CaselessEq<std::string> cmp;
+    const auto coord = layer->GetParamAsString(g_coordinate_transformation_mode, g_half_pixel);
+    const auto interpMode = layer->GetParamAsString(g_mode, g_linear);
+    InterpolateCoordTransMode coordinateTransMode = InterpolateCoordTransMode::HalfPixel;
+    InterpolateMode mode = InterpolateMode::Linear;
 
-    _stageBuilder->addInterpStage(model, layer->name, layer, layer->GetParamAsInt("align_corners", 0), inputs[0], outputs[0]);
+    coordinateTransMode = coordTransformModeMap.at(coord);
```

Comment:
`std::out_of_range` not a good exception for debug.
I think, it would be better to use `find` method and then `VPU_THROW_UNLESS(it != map.end())`

Reply:
done